shared/middleware/files_api.rb

```diff
@@ -476,6 +476,7 @@ def files_put_file(encrypted_channel_id, filename, body)
 
     bucket = FileBucket.new
     manifest = get_manifest(bucket, encrypted_channel_id)
+    manifest_is_unchanged = true
```

Comment:
Do the `copy_file` and `put_file` methods (indirectly) modify the manifest? If so, this PR is bunk.